packages/e2e/lib/e2e_driver.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:e2e/common.dart' as e2ecommon;
```

Comment:
I would stick to either e2e or common or just don't namespace the import, e2ecommon looks unusual to me.

Reply:
Thanks for the review! 

I'll use e2e then. 'Response' collides with another name. Hence I had to use a namespace.